jobs/k8s_claimer_deploy.groovy

```diff
@@ -60,29 +60,43 @@ job(name) {
     timestamps()
     colorizeOutput 'xterm'
     credentialsBinding {
+      string("KUBECONFIG_BASE64", "aa198b18-3566-438b-bab2-2dd169015567")
+      string("K8S_CLAIMER_SSH_KEY", "394d55f4-b9b6-4913-8b22-e874939f90b4")
+      string("GOOGLE_CLOUD_ACCOUNT_FILE", "ba7ab317-a820-4e70-9399-a54cf3a59949")
+      string("AUTH_TOKEN", "a62d7fe9-5b74-47e3-9aa5-2458ba32da52")
+      string("AZURE_SUBSCRIPTION_ID", "1b2376bb-38ed-480b-8bcc-81250ebaa327")
+      string("AZURE_CLIENT_ID", "862dbc6f-2fbe-4342-a797-c6433efb6761")
+      string("AZURE_CLIENT_SECRET", "3d5c3d60-8648-42f4-8401-7d96e08ca080")
+      string("AZURE_TENANT_ID", "528070f3-4799-4c1a-94d6-20a16177487a")
       string("GITHUB_ACCESS_TOKEN", defaults.github.credentialsID)
       string("SLACK_INCOMING_WEBHOOK_URL", defaults.slack.webhookURL)
       string("QUAY_PASSWORD", "8317a529-10f7-40b5-abd4-a42f242f22f0")
-      string("DEIS_URL", "1a72f795-58c5-4832-a463-28aa2765ba14")
-      string("DEIS_USERNAME", "ed8d00a4-5360-4182-8a86-c9175220296a")
-      string("DEIS_PASSWORD", "82b41d2d-53c6-4e24-bf77-108322ff799a")
     }
   }
 
   steps {
     shell '''
       #!/usr/bin/env bash
-
       set -eo pipefail
+      echo $KUBECONFIG_BASE64 | base64 --decode > kubeconfig
+
+      curl -f --silent --show-error --retry 5 --retry-delay 10 -O https://storage.googleapis.com/kubernetes-helm/helm-${defaults.helm.version}-linux-amd64.tar.gz
```

Comment:
if/when we want to interpolate groovy-land vars (like here w/ `defaults.helm.version`) we'll need to change the wrapping `shell` block to use double quotes instead of literal/single quotes (line 78 as of writing)

Note that once we do that, we need to escape all shell-land vars, e.g. `${PATH}` becomes `\${PATH}` so as to prevent groovy interpolation.